src/languages/pyret/PyretParser.ts

```diff
@@ -277,7 +277,7 @@ type ColumnSort = any;
 type ColumnSortOrder = any;
 type DefinedType = any;
 type DefinedValue = any;
-type Expr = ASTNode;
+type Expr = ASTNode & any;
```

Comment:
I find all these type aliases to be rather humorous. There really isn't a point to using typescript if all the types are aliased to `any`. The `Expr` type was the one exception in this list of type aliases, except that before `codemirror-blocks` exported types, `ASTNode` was also `any`.

anyway, the way that `Expr` gets used indicates that it is not in fact an `ASTNode`, but rather some derivation of ASTNode that hasn't been defined in the type system yet. So I've just set it back to `any` and maybe somebody who's familiar with the pyret ast data model can come up with a proper type later.